nengo_gui/static/lib/js/interact-1.2.4.js

```diff
@@ -3354,6 +3354,7 @@
         this.shiftKey  = event.shiftKey;
         this.metaKey   = event.metaKey;
         this.button    = event.button;
+        this.buttons   = event.buttons;
```

Comment:
``event.button`` is the mouse button that triggered the event. Since the ``ondragmove`` event is generated by ``interact.js`` as a response to the DOM ``MouseMoveEvent``, ``button`` does not have any meaning here. ``buttons`` on the other hand contains all mouse buttons that were pressed when the event was generated. This is what we want.

Regarding the more recent version of ``interact.js``: jap, let's use that instead. Should I upgrade ``interact.js`` in this PR or create another one?